flink-runtime/src/test/scala/org/apache/flink/runtime/jobmanager/JobManagerITCase.scala

```diff
@@ -609,6 +612,98 @@ WordSpecLike with Matchers with BeforeAndAfterAll {
         cluster.stop()
       }
     }
+
+    "remove execution graphs when the client ends the session explicitly" in {
```

Comment:
I think the consensus was to "lazily" move the `flink-runtime` package back to Java and write all new code and tests in Java. So for any additional tests that may come, can you write them in a new Java test class?